hal/armv8m/arch/cpu.h

```diff
@@ -0,0 +1,233 @@
+/*
+ * Phoenix-RTOS
+ *
+ * Operating system kernel
+ *
+ * CPU related routines
+ *
+ * Copyright 2014, 2017, 2022 Phoenix Systems
+ * Author: Jacek Popko, Pawel Pisarczyk, Aleksander Kaminski, Damian Loewnau
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#ifndef _HAL_ARMV8M_CPU_H_
+#define _HAL_ARMV8M_CPU_H_
+
+
+#if defined(CPU_NRF9160)
+#define CPU_NRF91
+#endif
+
+#include "types.h"
+
+#if defined CPU_NRF91
+#define SIZE_PAGE 0x200
+#endif
```

Comment:
Shouldn't be needed, will cause SIZE_PAGE to be not defined in `#else `case
```suggestion
#define SIZE_PAGE 0x200
```